css/src/tokens/colors.scss

```diff
@@ -5,6 +5,7 @@ $white-static: #fff;
 $black-static: #000;
 
 $text: var(--theme-text);
+$text-fluent: var(--theme-text-fluent);
```

Comment:
I think we should have minimal, if any, changes in this file. For example, there should be no `$text-fluent` only `$text`.